packages/embark/src/lib/constants.json

```diff
@@ -88,6 +88,8 @@
     "dir": "config",
     "blockchain": "blockchain.json",
     "storage": "storage.json",
-    "communication": "communication.json"
+    "communication": "communication.json",
+    "embarkjs": "embarkjs.js",
+    "contractsJs": "contracts"
```

Comment:
I assume this is describing the **path** to a configuration file/folder, so maybe we can call this sth like `contractsConfigLocation`